flight/Modules/Actuator/actuator.c

```diff
@@ -462,7 +462,7 @@ static void actuator_task(void* parameters)
 		bool success = true;
```

Comment:
Should remove this too since it no longer does anything. Can probably also remove ActuatorCommand.NumFailedUpdates?